### DISCUSSION TITLE
change de coordinates of the IFC model (including rotation)

### DISCUSSION THREAD

**Discussion by klhonda** [IfcOpenShell/IfcOpenShell#3375] [Q&A]
Hello guys!

I have an IFC model of a building that I need to locate in 5 different sites (5 different federated models), is that possible to change the coordinates of the base point of this model thru ifcopenshel? I’d like to automatizate this process thru a python code, otherwise I’ll have to do the whole process everytime that I review my model.

**Comment by Moult** [comment]
Yes, use IfcPatch: https://blenderbim.org/docs-python/ifcpatch.html

Use the OffsetObjectPlacements recipe: https://blenderbim.org/docs-python/autoapi/ifcpatch/recipes/OffsetObjectPlacements/index.html
